File: frogment/src/main/java/ch/grze/frogment/SwitchAware.java
package ch.grze.frogment;

import ch.grze.frogment.activity.FrogmentActivityData;
import ch.grze.frogment.frogment.FrogmentData;

public interface SwitchAware {
    void switchFrogment(FrogmentData data);
    void switchActivity(FrogmentActivityData data);
}

File: frogment/src/main/java/ch/grze/frogment/SwitchAware.kt
package ch.grze.frogment

import ch.grze.frogment.activity.FrogmentActivityData
import ch.grze.frogment.frogment.FrogmentData

interface SwitchAware {
    fun switchFrogment(data: FrogmentData)
    fun switchActivity(data: FrogmentActivityData)
}
